### ISSUE TITLE
Collective URL with Slash

### ISSUE THREAD

**Issue by l0gicgate** [opencollective/opencollective#2084]
**Describe the bug**

**To Reproduce**
1. Go to create a new collective
2. Pick Open Source Project
3. Click on Get Started
4. Select the repository you want to create a collective for
5. Enter the collective name
6. Enter the collective URL but include a slash in it e.g.: `your-org/your-collective`
7. Click on Create Collective
8. Navigate to ```http://www.opencollective.com/your-org/your-collective``` to find that it is not found. No way to edit it either.

**Expected behavior**
Create an opencollective url that would be sub-directoried to your org.
```http://www.opencollective.com/your-org/your-collective```

**Screenshots**
![ss+(2019-06-03+at+10 26 00)](https://user-images.githubusercontent.com/6510935/58818066-70a1bc80-85ea-11e9-851f-df9ebeb3c48a.png)
![ss+(2019-06-03+at+10 29 42)](https://user-images.githubusercontent.com/6510935/58818119-8d3df480-85ea-11e9-81a8-2c04634d95e2.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version 74.0.3729.169

**Comment by znarf** [comment]
Thank you for reporting that. We're working on this, will be fixed soon. Closing as duplicate of https://github.com/opencollective/opencollective/issues/2077

In the meantime, please contact support if you need to be unblocked.

**Comment by l0gicgate** [comment]
Anything that we can do in the meantime? Can you delete that collective so I can re-create it?

**Comment by znarf** [comment]
We can do support here also :-) Should we edit it to be `slimphp` or `slim`?

**Comment by l0gicgate** [comment]
`slim` is fine thanks!

**Comment by znarf** [comment]
Done http://opencollective.com/slim

Looks like you also created an Organization and a User account with the same name. If it was by accident, maybe you want to rename them or delete them.

**Comment by l0gicgate** [comment]
I'm sorry if these are noob questions but I thought an organization could have collectives. I just followed the regular process when signing up. Do I need a collective? An org? Both?

**Comment by znarf** [comment]
Normally, you just need one Collective "Slim Framework", whatever the number of repositories. It doesn't make sense to split into multiple collectives.

An Organization at Open Collective is different from the GitHub definition. It's mostly used by companies donating to Collectives. It can be also be used to host (as in fiscal hosting) Collectives.

Finally, the User account is usually under your real name, to show who is administrating and contribution to the Collective.

More info here https://docs.opencollective.com/ or contact support https://opencollective.com/support.

**Comment by l0gicgate** [comment]
Okay gotcha, then in this case I will change my user to be myself, delete the organization and can we change the collective to be `slimphp` instead of Slim? 

Thank you.
